core/appeal/service.go

```diff
@@ -349,19 +396,27 @@ func (s *Service) MakeAction(approvalAction domain.ApprovalAction) (*domain.Appe
 			}
 			if len(notifications) > 0 {
 				if err := s.notifier.Notify(notifications); err != nil {
-					s.logger.Error(err.Error())
+					s.logger.Error("failed to send notifications", "error", err.Error())
 				}
 			}
 
+			auditKey := AuditKeyApprove
+			if approvalAction.Action == "revoke" {
```

Comment:
should we make `const` out of all possible actions and store it in a central place instead of hardcoding it here?